custom_components/media_player_template/media_player.py

```diff
@@ -109,9 +109,10 @@
         vol.Optional(CONF_ENTITY_PICTURE_TEMPLATE): cv.template,
         vol.Optional(CONF_AVAILABILITY_TEMPLATE): cv.template,
         vol.Optional(CURRENT_SOURCE_TEMPLATE): cv.template,
-        vol.Optional(ON_ACTION): cv.SCRIPT_SCHEMA,
-        vol.Optional(OFF_ACTION): cv.SCRIPT_SCHEMA,
+        vol.Required(ON_ACTION): cv.SCRIPT_SCHEMA,
+        vol.Required(OFF_ACTION): cv.SCRIPT_SCHEMA,
```

Comment:
Should indeed be reverted
```suggestion
        vol.Optional(ON_ACTION): cv.SCRIPT_SCHEMA,
        vol.Optional(OFF_ACTION): cv.SCRIPT_SCHEMA,
```